gradle/publish.gradle

```diff
@@ -129,11 +163,16 @@ task publishGroovydoc {
 task publishRelease {
     description = "Publish release for plugin version: $tag"
     group = 'release'
-    if (project.hasProperty('dryRun') && project['dryRun'] == 'false') {
-        dependsOn prepareRelease, publishArtifact, publishGroovydoc, publishPlugins
-        doLast {
-            grgit.push {
-                tags = true
+    boolean isDryRun = buildProperties.cli['dryRun'].or(true).boolean
+    if (!isDryRun) {
```

Comment:
Can we invert? `if (isDryRun) { ... } else { ... }`

Reply:
42b007692d32051a77eaee87ed686276c8acc813